### ISSUE TITLE
GREI 5: Task 6 - Assemble and provide metrics about HDV data collection for therapeutic areas

### ISSUE THREAD

**Issue by cmbz** [IQSS/dataverse-pm#229]
## Overview

- Assemble and provide metrics about HDV data collection for therapeutic areas
- Therapeutic areas include: `Cardiology`, `Systems Neuroscience`, `Quality of Life`, `Bioinformatics Software`, and `Real-Time Polymerase Chain Reaction`

## Tasks

- [x] Identify HDV source for information about therapeutic areas (e.g., Dataverse subjects and keywords)
- [x] Map existing HDV information to therapeutic areas (e.g., specific keywords may map to a therapeutic area)
- [x] Collect metrics about HDV datasets exhibiting related keywords and subjects
- [x] Share metrics in designated [GREI spreadsheet](https://docs.google.com/spreadsheets/d/1NneOwZrZqyiQxej1XnXehFURRDpeO0OQLo9kzf7vEsc/edit#gid=0) 
- [ ] Consider approaches to operationalize this reporting on a regular basis (if needed) 

## Resources
- https://github.com/IQSS/dataverse-pm/issues/124
- https://github.com/IQSS/dataverse-pm/issues/118

**Comment by cmbz** [comment]
## Status: May 2024
- TBD

**Comment by jggautier** [comment]
@sbarbosadataverse, I searched for datasets whose metadata contains any of the list of categories that other GREI repositories have listed in [a Top Biomedical Research Categories in GREI Repository Holdings spreadsheet](https://docs.google.com/spreadsheets/d/1NneOwZrZqyiQxej1XnXehFURRDpeO0OQLo9kzf7vEsc).

Could you review [the datasets from that search](https://docs.google.com/spreadsheets/d/1OAQiSkgyeb_YdM4rFhl439FUeNadvmg5R5r0d4PN4us/edit#gid=0) to see if many are relevant, if any stick out as irrelevant, and to give a sense of how many might be missing just based on the list of 10 terms?

This will help me evaluate the way I'm finding these datasets, before we consider using more search terms like we spoke about, such as the "therapeutic areas" at https://www.cdisc.org/standards/therapeutic-areas/disease-area and [the names of NIH centers and institutes](https://www.nih.gov/institutes-nih/list-institutes-centers).

We also spoke about looking at lists of keywords and topic classifications in the metadata of datasets from NIH-funded research in the Harvard Dataverse Repository (https://github.com/IQSS/dataverse-pm/issues/217).

Could you take a look at the [keywords](https://docs.google.com/spreadsheets/d/1OAQiSkgyeb_YdM4rFhl439FUeNadvmg5R5r0d4PN4us/edit#gid=1917325999) and [topic classifications](https://docs.google.com/spreadsheets/d/1OAQiSkgyeb_YdM4rFhl439FUeNadvmg5R5r0d4PN4us/edit#gid=1477983457)?

My impression is that someone with domain knowledge would need to review these before we can use them for searching.

**Comment by cmbz** [comment]
## Status: June 2024
- Pending

**Comment by jggautier** [comment]
@sbarbosadataverse and @cmbz, I'm going to close this GitHub issue. I'm curious how these counts will be used and plan to ask about them during the GREI-Monthly CWG Meeting on July 3.

**Comment by jggautier** [comment]
Re-opening this issue. @sbarbosadataverse asked that I include counts of datasets that include the term "covid19"in their metadata. I'm getting that count now and will update [the Harvard Dataverse tab of the spreadsheet](https://docs.google.com/spreadsheets/d/1NneOwZrZqyiQxej1XnXehFURRDpeO0OQLo9kzf7vEsc/edit?gid=1430356311#gid=1430356311) today.

**Comment by jggautier** [comment]
I updated [the "Harvard Dataverse" tab](https://docs.google.com/spreadsheets/d/1NneOwZrZqyiQxej1XnXehFURRDpeO0OQLo9kzf7vEsc/edit?gid=1430356311#gid=1430356311) and [the "Aggregate" tab](https://docs.google.com/spreadsheets/d/1NneOwZrZqyiQxej1XnXehFURRDpeO0OQLo9kzf7vEsc/edit?gid=1821025208#gid=1821025208) of the spreadsheet.

**Comment by sbarbosadataverse** [comment]
Closed the last remaining checkbox as we consider in a new issue how to make use of the information we collected for this ask from the GREI planning unit
